packages/builder/src/pages/builder/auth/reset.svelte

```diff
@@ -37,47 +49,92 @@
     } catch (error) {
       notifications.error("Error getting org config")
     }
+    loaded = true
   })
 </script>
 
-<div class="login">
-  <div class="main">
-    <Layout>
-      <Layout noPadding justifyItems="center">
-        <img src={$organisation.logoUrl || Logo} alt="Organisation logo" />
-      </Layout>
-      <Layout gap="XS" noPadding>
-        <Heading textAlign="center">Reset your password</Heading>
-        <Body size="S" textAlign="center">
-          Please enter the new password you'd like to use.
-        </Body>
-        <PasswordRepeatInput bind:password bind:error />
-      </Layout>
+<TestimonialPage>
+  <Layout gap="S" noPadding>
+    {#if loaded}
+      <img alt="logo" src={$organisation.logoUrl || Logo} />
+    {/if}
+    <Layout gap="XS" noPadding>
+      <Heading size="M">Reset your password</Heading>
+      <Body size="M">Please enter the new password you'd like to use.</Body>
+    </Layout>
+
+    <Layout gap="S" noPadding>
+      <FancyForm bind:this={form}>
+        <FancyInput
+          label="Password"
+          value={formData.password}
+          type="password"
+          on:change={e => {
+            formData = {
+              ...formData,
+              password: e.detail,
+            }
+          }}
+          validate={() => {
+            let fieldError = {}
```

Comment:
Confirmation passwords are a bit of a different case so it's probably fine to use custom errors here, but again you probably don't need to.